### DISCUSSION TITLE
creating a fieldset with POP

### DISCUSSION THREAD

**Discussion by norafried** [Parcels-code/Parcels#1249] [Q&A]
I'm trying to create a fieldset with POP but I have problems with the B-Grid that POP has. I tried to follow the indexing example and adjust that to POP using from_pop (https://nbviewer.org/github/OceanParcels/parcels/blob/master/parcels/examples/documentation_indexing.ipynb) but I keep having issues with that. it always returns:

ValueError: On a B-grid, the dimensions of velocities should be the (top) corners of the grid cells, so the same for U, V and W

Is there an example for POP?

**Comment by erikvansebille** [comment]
@pdnooteboom might have some code to help you, as he has worked with pop data in his PhD work. Or you may have a look at the code of [this paper](https://doi.org/10.1371/journal.pone.0238650) at https://github.com/pdnooteboom/PO_res_error

**Reply by pdnooteboom** [reply]
Yes, you could check out specifically [this part of the code](https://github.com/pdnooteboom/PO_res_error/blob/master/low_spatial_res/setfieldsets.py#L55), which uses a curvilinear grid with 1 degree horizontal resolution, or [this part](https://github.com/pdnooteboom/PO_res_error/blob/master/high_spatial_res/atsf_cart_POP_0.1res.py#L150), which uses 0.1 degree horizontal resolution, tripolar grid with partial bottom cells. 
I hope this helps.

**Reply by norafried** [reply]
Great, thanks. I think I found (one of) the errors. I can load the fieldset for one time step now. I have one file per time step containing all the variables. I tried to load them the same was as in the NEMO 3D tutorial. Not sure why that doesn't work. Is there any additional trick? Thanks for all your help!
